docs/refs/api/rtos/callbacks.md

```diff
@@ -353,7 +243,7 @@ void callmemaybe(Callback<void(float)> cb) {
 }
 ```
 
-The Callback class is what’s known in C++ as a “Concrete Type”, that is, the Callback class is lightweight enough to be passed around like an int, pointer, or other primitive type.
+The Callback class is a “Concrete Type”. That is, the Callback class is lightweight enough to be passed around like an int, pointer or other primitive type.
```

Comment:
Is there a way we can emphasize the "Concrete Type" is a C++ term? This scoping is important if the reader wants to google for "Concrete Type" since the term has different meanins in different languages.